concert/app/test/__init__.py

```diff
@@ -1,6 +0,0 @@
-import os
-
-# setup dummy environment variables
-os.environ["AWS_DEFAULT_REGION"] = "us-east-1"
```

Comment:
This one can stay here, it's not specific to any one function and is needed to keep boto3 from being a little bitch